app/controllers/api/api_error.rb

```diff
@@ -0,0 +1,6 @@
+class API::APIError
```

Comment:
Joseph, can you explain where to put classes that represent entities passed around and never persisted to the DB like this one? It certainly doesn't fit in models and doesn't seem to fit in controller either. Should this be a view? I don't get Rails organization

Reply:
Oh wow, this is quite different from other languages. Thank you!

Reply:
Where would you put the file if it was referenced in other directories? app/lib?